### DISCUSSION TITLE
How to add only 2 buttons (sequential)

### DISCUSSION THREAD

**Discussion by Tripiro** [azzajess/USB-Shifter-to-Thrustmaster-Wheelbase#8] [General]
What would be the connections to put only sequential?

I am making a sequential (only 2 buttons) with the 3D printer and I wanted to know which buttons to use.

 In the arduino pro micro soldier:
Gear 3 and 4?
Do I have to enable pin 10 (Sequential Shift)?

Thank you,

Excuse my English, I'm Spanish

**Comment by azzajess** [comment]
Without modifying any code it's pin 3 and 4 And you ground pin 10 on Arduino.
Let me know how that goes. If you have issues, I might reconfigure the code for seq only if need be

**Comment by Tripiro** [comment]
Thanks a lot. Testing the arduino in the wheel. Grounded pin for sequential and pin 4 and 5. Working fine.

**Comment by stivon77-lgtm** [comment]
Hello, I would like to carry out the project of creating a sequential lever using the code provided with an Arduino nano. Is it compatible?

**Reply by azzajess** [reply]
I'm not 100% but if arduino nano can use the same pin out and logic then yes, but will still need to be 3v3 compatible. Looking up some boards can have their step down disables so 3v3 can be run directly, but you will need check for your board
But basically following the above mentions, connecting gear up and down buttons to pin 4 and 5 and then grounding pin 10 (which enables sequential mode)
Don't see why not?
